edgelessdb/getting-started/recovery.md

```diff
@@ -0,0 +1,38 @@
+
+# Recovery
+
+## Information
+!> Recovery is only available when EdgelessDB is running standalone, and only when a recovery key has been defined in the manifest. When used with MarbleRun, consider using its [recovery feature](https://docs.edgeless.systems/marblerun/#/content/features/recovery).
+
+Persistent storage for confidential applications requires a bit of attention.
+By design, SGX sealing keys are unique to a single CPU, which means using the default SGX sealing methods has some caveats.
+For example, sealing data while running on one host could mean the data can't be unsealed when running on another host later on.
+
+EdgelessDB makes persistence straightforward by using a so-called *virtual sealing key*, which is a randomly generated encryption key which is stored on disk with the help of the SGX sealing key. However, when EdgelessDB is moved to another physical host, a manual step is required to ensure the database can be recovered.
+Therefore, [the manifest allows for specifying a designated *Recovery Key*](reference/manifest.md). The Recovery Key is a public RSA key. During the initial upload of the manifest, EdgelessDB generates its symmetric state-encryption key and returns an Base64 encoded, RSA encrypted version of it using the public key specified in the manifest.
```

Comment:
```suggestion
EdgelessDB makes persistence straightforward by using a so-called *virtual sealing key*, which is a cryptographically securely generated encryption key used for sealing data to persistent storage. While being scheduled on the same CPU, EdgelessDB restarts autonomously. However, when EdgelessDB is moved to another physical host, the advantage of the *virtual sealing key* becomes evident.  Because we can securely backup the virtual sealing key to a trusted user (not possible with the real SGX sealing key), the trusted user can recover EdgelessDB later on.
To that end, [the manifest allows for specifying a designated *Recovery Key*](reference/manifest.md). The Recovery Key is a public RSA key. During the initial upload of the manifest, EdgelessDB generates its virtual sealing key and returns it RSA encrypted with the public key specified in the manifest.
```